### PR TITLE
Added new question for HR

### PR DESCRIPTION
**Pull request by Kathuria**

### CODE REVIEW

**Comment by sanjeevyadavIT** [comment]
hi @Kathuria  Thank you so much for the your contribution 🎉

File: README.md [hunk 1]
```diff
@@ -19,6 +19,7 @@ At the time of joining, the HR potrays the company like it is match made in Heav
 | 10 | Usually help half or whole financially for IT events and certifications?|
 | 11 | Can you describe the culture of the company?|
 | 12 | How easily the upper management is approachable for suggestion or queries?|
+| xx | Does your organisation is open for on-site opputunities and Can I get this oppurtunity on starting of day of an organisation? |
```
Comment (sanjeevyadavIT) [inline README.md]: - In place of xx, mention the question number in chronological order
- their is a typo in spelling "opportunity", please fix that
- Second part of the question could be modified to `if yes, do you give these opportunity to new comers`?

**Review by sanjeevyadavIT — CHANGES_REQUESTED** [review]
There are some changes requested, please look into it

and if you are having trouble, making new changes, let me know, would be happy to help :smile:

File: README.md (hunk 1 above)
Comment (Kathuria) [reply README.md]: Thanks for the update. Changes are Done, Please Review it again

**Pull request merged**